### ISSUE TITLE
vehicles can't drive after completing certain faction camp jobs

### ISSUE THREAD

**Issue by kingluan612** [CleverRaven/Cataclysm-DDA#46840]
# Describe the bug

After completing the build northeast shack/tent job, vehicles directly to the left of the bulletin board give the message that they are too heavy for their engines when trying to drive, and using ^ says that there are controls found on places where the vehicle can be controlled. Only    e -> v -> e    allows the character to stop controlling their vehicle.

# Steps To Reproduce

Make a world 
Create a field faction camp, and get an NPC to do one of the build shack jobs
Create a vehicle and place it to the left of the bulletin board
Recall NPC from faction camp job after they complete it
Try to drive the vehicle

(optional for more weirdness)
Grab the vehicle and try to drag it around, sometimes the vehicle will move and everything is fixed, sometimes things really mess up.

# Expected behavior

Vehicles to the right/below/above the bulletin board work fine after recalling an NPC from one of the jobs, its just ones to the left that are broken. Emergency recalling also doesn't break the vehicle (but also removes all NPC jobs in the faction camp so its a dud either way)

# Screenshots

![image](https://user-images.githubusercontent.com/67880607/104857059-ff5c6d80-58e3-11eb-8d66-1bdd17262fa9.png)

Other weirdness with trying to interact with the vehicle:
![image](https://user-images.githubusercontent.com/67880607/104857092-2d41b200-58e4-11eb-8f41-f3b87a36ff85.png)
![image](https://user-images.githubusercontent.com/67880607/104857339-d5a44600-58e5-11eb-8440-67648e06b392.png)

# Versions and configuration

- OS: Windows
    - OS Version: 10.0 1909
- Game Version: 0.E-9017-gac88c03-dirty [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Bionic Professions [package_bionic_professions]
]

# Additional context

NPCs will not follow you to the vehicle if you try to drive it, instead they run away

[Hovland - Copy.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/5826911/Hovland.-.Copy.zip)
In this save, interact with the bulletin board and recall the NPC (not emergency recall). Then try to drive the bike to the left. Move the bike around to different positions before recalling NPC to test if it can move or not after recalling.

**Comment by PaulBlay** [comment]
Probably not related, but I recently had my character's custom motorcycle 'too heavy for their engines' despite nothing being obviously wrong. I stopped the engine, restarted, and it worked again. It has not reoccurred.

**Comment by kingluan612** [comment]
Yeah, reloading the save and even dragging the vehicle in the same direction as the side thats grabbed fixes this, but it was confusing to me the first time this happened because I couldn't dismount off my bike, but it also couldn't move either.

**Comment by PaulBlay** [comment]
I think this happens when you begin starting the engine, but get interrupted. You are left in a state where the engine is not engaged but the vehicle is officially being operated. The ^ dialog does not have an option to 'start the engine' so all you can do is stop driving.

**Comment by OliveChukar** [comment]
Just tested in 0.E-9140-gedd42fc. Got the same problem with provided save and a fresh save with Dark Days Ahead [dda] only. On the fresh save I used a flatbed truck and when it was in the stuck state with me driving the (e)xamine vehicle screen did not show any fuel consumption or battery drain. This supports the failure to start engine idea however when trying to drive there were no interruption messages.

**Comment by actual-nh** [comment]
Are there any vehicles with racks around the camp? (Am thinking of #46794.)

**Comment by OliveChukar** [comment]
Just checked provided save and the only vehicle is the bike that gets bugged and fails to move. My test save did have an APC but it did not have bike racks and when I moved it away the bug still happened. If I understand correctly #46794 was a crash and the faction camp thing was unrelated. This issue does not cause a crash for me(46840).
